File: app/src/main/java/org/software/learning/model/common/ResponseTokens.java
package org.software.learning.model.common;

public class ResponseTokens {
    public static ResponseToken validateError = new ResponseToken(400, "validateError", "验证错误");
    public static ResponseToken ok = new ResponseToken(200, "success", "操作成功");
    public static class Login {
        public static ResponseToken registered = new ResponseToken(1, "registered", "当前用户已经被注册");
        public static ResponseToken noUser = new ResponseToken(2, "no_user", "不存在该用户");
        public static ResponseToken passwordError = new ResponseToken(3, "passwordError", "密码错误");
        public static ResponseToken noToken = new ResponseToken(4, "no_token", "不存在此token");
        public static ResponseToken tokenExpired = new ResponseToken(5, "token_expired", "token已过期");
    }
}

File: server/src/main/java/org/learning/server/model/common/ResponseTokens.kt
package org.learning.server.model.common

object ResponseTokens {
    val ok = ResponseToken(200, "ok", "ok")
    val validateError = ResponseToken(400, "validateError", "validateError")
    val failed = ResponseToken(403, "failed", "failed")

    object User {
        val noLogin = ResponseToken(1, "noLogin", "未登录无法访问该资源")
        var forbidden = ResponseToken(401, "forbidden", "没有足够的权限")
    }
}